packages/slate-react/src/plugins/ANDROID.md

```diff
@@ -0,0 +1,219 @@
+# Android
+
+The following is a list of unexpected behaviors in Android
+
+# Debugging
+
+```
+slate:android,slate:before,slate:after,slate:update,slate:reconcile
+```
+
+# API 25
+
+### Backspace Handling
+
```

Comment:
"There appears to be no way to discern that backspace was placed in the events."

Reply:
Thanks. I'll update that text. Should have said something like: There is no way to tell from the event that there was a backspace in API 25. I've got a working DOM analysis tool that will help me determine when a node has been obliterated which indicates that a backspace occurred over a critical threshold like deleting a block or a span that includes a mark. In other cases like a regular character deletion, we the reconcile code handles it properly.